core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -8,22 +8,27 @@
               <label for="explorationTitle" class="col-lg-2 col-md-2 col-sm-2">Title</label>
               <div class="col-lg-10 col-md-10 col-sm-10">
-                <input id="explorationTitle" type="text" class="form-control protractor-test-exploration-title-input" ng-model="explorationTitleService.displayed" ng-blur="saveExplorationTitle()" placeholder="Choose a title for your exploration." focus-on="<[::EXPLORATION_TITLE_INPUT_FOCUS_LABEL]>" maxlength="40">
+                <input id="explorationTitle" type="text" class="form-control protractor-test-exploration-title-input" ng-model="explorationTitleService.displayed" ng-blur="saveExplorationTitle()" placeholder="Choose a title for your exploration." focus-on="<[::EXPLORATION_TITLE_INPUT_FOCUS_LABEL]>" maxlength="40" data-ng-trim="false">
                 <span class="help-block" style="font-size: smaller">
                   <em>Please use at most 40 characters, so that this fits in the summary card.</em>
+                  <span ng-if="explorationTitleService.displayed.length > 0">
+                  <em><[ "Characters used: " +  (explorationTitleService.displayed.length) + "/40" ]></em>
+                  </span>
                 </span>
               </div>
             </div>
-            <div class="form-group" ng-class="{'has-error': !explorationObjectiveService.displayed || explorationObjectiveService.displayed.length < 15}">
+            <div class="form-group" ng-class="{'has-error': !explorationObjectiveService.displayed || ((explorationObjectiveService.displayed.length < 15) || (explorationObjectiveService.displayed.length > 100))}">
               <label for="explorationObjective" class="col-lg-2 col-md-2 col-sm-2">Goal</label>
               <div class="col-lg-10 col-md-10 col-sm-10">
                 <input id="explorationObjective" type="text"
                        class="form-control protractor-test-exploration-objective-input"
                        ng-model="explorationObjectiveService.displayed"
-                       ng-blur="saveExplorationObjective()" placeholder="Learn how to ...">
-                <span class="help-block" style="font-size: smaller">
-                  What does this exploration help people to do?
-                  <span ng-if="!explorationObjectiveService.displayed || explorationObjectiveService.displayed.length < 15">Please provide a complete, descriptive sentence.</span>
+                       ng-blur="saveExplorationObjective()" placeholder="Learn how to..." data-ng-trim="false">
+                <span class="help-block" style="font-size: smaller"> 
+                    <em>In a complete sentence, tell people what they'll learn from this exploration.</em>
+                    <span ng-if="explorationObjectiveService.displayed.length > 0">
+                    <em><[ "Characters used: " +  (explorationObjectiveService.displayed.length) + "/100" ]></em>
```

Comment:
I don't understand your question. But this em tag is contained within the span tag on the previous line, if I understand correctly. Since it is a child of span, and not a sibling, it should be indented by 2.